byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +60,33 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
     return place;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query allUsers = new Query("sample");
+    PreparedQuery preparedUsers = datastore.prepare(allUsers);
+    for(Entity enteredUser: preparedUsers.asIterable()) {
+      long id = (long) enteredUser.getProperty("id");
+      if(id == 2452) {
```

Comment:
Same comment as above. You can limit your query to just the object with this specific id

Reply:
Fixed